player/js/module_worker.js

```diff
@@ -58,11 +58,11 @@ var lottiejs = (function(window) {
 
 var animations = [];
 
-var onmessage = function(evt) {
+onmessage = function(evt) {
```

Comment:
`var onmessage` is just a variable with function expression inside of it. But for Worker we've to initialize event as [it is](https://developer.mozilla.org/en-US/docs/Web/API/Worker/onmessage).